src/stdlib/numbers/numbers.verona

```diff
@@ -0,0 +1,18 @@
+// Copyright Microsoft and Project Verona Contributors.
+// SPDX-License-Identifier: MIT
+class I8 {}
```

Comment:
So, numeric types are markers for code generation only, with storage size and signal defined by their names, location and duration defined by their region, right?

Reply:
Yeah exactly. The representation of a numeric type is _not_ a pointer, but a concrete machine word. What we'll do is make sure that numbers (and bools) are always `imm`, and then the type system "just works".